src/app/globals.css

```
@@ -1,104 +1,193 @@
+  --color-ring: #8a8a8a;
+  --color-primary: #090909;
+  --color-primary-foreground: #ffffff;
+  --color-secondary: #f7f7f7;
+  --color-secondary-foreground: #090909;
+  --color-destructive: #e53e3e;
 
+  /* Fonts */
+  --font-sans: var(--font-pretendard);
+  --font-serif: var(--font-nanum-myeongjo);
+
+  /* Border radius 임시*/
```

Stylelint (17.9.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)